app/components/speaker-list.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+const { Component } = Ember;
+
+export default Component.extend({
+  speakers: [{  name: 'Speaker 1', organisation: 'Org 1' },
```

Comment:
Try to understand ember by going through the docs this week ... Then during the intro call, I'll try to give a short over-view of how things work in ember to everyone.